website/source/docs/enterprise/runs/variables-and-configuration.html.md

```diff
@@ -75,9 +90,44 @@ For any of the `GITHUB_` attributes, the value of the environment variable will
 be the empty string (`""`) if the resource is not connected to GitHub or if the
 resource was created outside of GitHub (like using `terraform push`).
 
+### Personal Variables
+
+Personal variables can be created at the Environment or Organization level. All
+Personal variables are private and scoped to the user that created them. Just
+like shared Environment variables, they are injected into the virtual 
+environment during the `plan` and `apply` phases.
+
```

Comment:
Personal Environment variables are scoped to just the environment they are attached to, while Personal Organization variables are applied across any environment a user triggers a Terraform run in.